src/examples/dialog/dialog-form.tsx

```diff
@@ -34,10 +44,21 @@ export class DialogFormExample {
         this.closeConfirmation = this.closeConfirmation.bind(this);
         this.onConfirmPositive = this.onConfirmPositive.bind(this);
         this.onConfirmNegative = this.onConfirmNegative.bind(this);
+        
+        this.scrollStop=this.scrollStop.bind(this);
     }
 
     public render() {
         return [
+            <limel-input-field
+                label="Number Field Label"
+                value={this.value}
+                type="number"
+                formatNumber={this.formatNumber}
+
+                onWheel={() => { this.el.blur() }}
```

Comment:
`this.el` here will be the entire example component, not just the input field, which means we are trying to blur the whole component. Is that what we want? :thinking: